caluma/extensions/permissions.py

```diff
@@ -1,6 +1,28 @@
-from caluma.caluma_core.permissions import BasePermission
+from caluma.caluma_core.mutation import Mutation
+from caluma.caluma_core.permissions import (
+    BasePermission,
+    object_permission_for,
+    permission_for,
+)
 
 
 class MySAGWPermission(BasePermission):
-    def has_permission(self, mutation, info):
-        return True
+    def _admin_or_sagw(self, info):
+        groups = info.context.user.groups
+        return "admin" in groups or "sagw" in groups
+
+    def _admin_or_sagw_or_own(self, info, instance):
+        groups = info.context.user.groups
+        return (
```

Comment:
I'd probably combine this with the above method at the call-site to avoid double-implementation.

Reply:
Done.